totest-manager.py

```diff
@@ -435,7 +493,28 @@ def _release_package(self, project, package, set_release=None):
         else:
             self.api.retried_POST(url)
 
-    def _release(self, set_release=None):
+    def _release_docker_image(self, pkg, snapshot, repo_url, primary_tree):
+        """The docker image is wrapped in an RPM in the ToTest repo"""
+
+        # Download the RPM to a temporary location
+        rpm_path = self._download_snapshot_rpm(pkg['pkg'], repo_url, primary_tree)
+        # Create a temporary dir for extraction
```

Comment:
nit: a summary comment about the overall process makes sense, but commenting every line would seem to indicate either the code needs to be refactored or unnecessary.